engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -195,13 +204,25 @@ private void initRenderingSupport() {
         shaderManager.initShaders();
         postProcessor.initializeMaterials();
         initMaterials();
+
+        context.put(WorldRenderer.class, this);
```

Comment:
At this stage, every node depends on `WorldRendererImpl`, wouldn't be better to not make constructors overly crowded?